about/index.md

```diff
@@ -1,44 +1,13 @@
-	<a href="http://twitter.com/UpdayDevs" title="Alex"><img src="https://github.com/joker54.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Josiane"><img src="https://github.com/jomilanez.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Jona"><img src="https://github.com/jonadinges.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Kavya"><img src="https://github.com/kavyaShreeHS.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Andi"><img src="https://github.com/krnki.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Tomasz"><img src="https://github.com/ktomek.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Lennard"><img src="https://github.com/lenn4rd.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Lucia"><img src="https://github.com/luciapayo.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Maciej"><img src="https://github.com/maciejwalkowiak.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Maria"><img src="https://github.com/mfernandezpajares.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Timo"><img src="https://github.com/ulich.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Tino"><img src="https://github.com/noacktino.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Pablo"><img src="https://github.com/pabloqc.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Peter"><img src="https://github.com/peter-tackage.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Peter"><img src="https://github.com/pkrauss-asideas.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Robert"><img src="https://github.com/rbordo.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Artur"><img src="https://github.com/sevos.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Simone"><img src="https://github.com/SimoneDzapoGarcia.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Nicola"><img src="https://github.com/sirnicolaz.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Mike"><img src="https://github.com/suxor42.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Richard"><img src="https://github.com/sweatyrichard.png" alt="image"></a>
-	<a href="http://twitter.com/UpdayDevs" title="Will"><img src="https://github.com/will-nolan.png" alt="image"></a>
+    {% assign devs = site.data.developers | sort: 'name' %}
+    {% for developer in devs %}
+	    <a href="http://twitter.com/{{ if developer.twitter }}{{ developer.twitter}}{{ else }}UpdayDevs{{ endif }}" title="{{ developer.name }}"><img src="https://github.com/{{ developer.github }}.png" alt="image"></a>
```

Comment:
@maciejwalkowiak some how the if clause is buggy - it is resolved e.g. for me with my twitter name + UpdayDevs
but I don't see the error...